node/sc/bridge_accounts.go

```diff
@@ -70,8 +81,32 @@ func (ba *BridgeAccounts) GetBridgeOperators() map[string]interface{} {
 	return res
 }
 
+// GetParentOperatorFeePayer can return the fee payer of parent operator.
+func (ba *BridgeAccounts) GetParentOperatorFeePayer() common.Address {
+	return ba.pAccount.feePayer
+}
+
+// SetParentOperatorFeePayer can set the fee payer of parent operator.
+func (ba *BridgeAccounts) SetParentOperatorFeePayer(feePayer common.Address) error {
+	ba.pAccount.feePayer = feePayer
+	ba.db.WriteParentOperatorFeePayer(feePayer)
+	return nil
+}
+
+// GetChildOperatorFeePayer can return the fee payer of child operator.
+func (ba *BridgeAccounts) GetChildOperatorFeePayer() common.Address {
```

Comment:
When is `GetChildOperatorFeePayer` used?

Reply:
This will be used for value transfering.
But for now, we don't use this. :)

Reply:
What does "for now, we don't use this" mean? 
Will it be used in upcoming PR or not used in the near future?

Reply:
It will be implemented in the next release, not this release.